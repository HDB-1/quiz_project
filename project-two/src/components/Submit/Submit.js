import React from 'react'
import './Submit.css'
export default function Submit(props) {
    return (
        <div className='submit'>
            <button id='skipBtn' onClick={props.skip}>Skip</button>
            <button id='submitBtn' onClick={() => props.submit(props.answerSelected)}>Submit</button>
        </div>
    )
}
