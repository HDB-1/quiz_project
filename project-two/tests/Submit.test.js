import React from 'react';
import { shallow } from 'enzyme';
import Submit from '../src/components/Submit/Submit'

describe('Shallow Submit', () => {
  let wrapper;
  
  beforeEach(() => wrapper = shallow(<Submit />)); // this automatically tests for rendering without crashing.
  it("Should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
