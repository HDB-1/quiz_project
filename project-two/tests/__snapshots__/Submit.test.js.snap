// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Submit Should render correctly 1`] = `<div />`;
